_posts/2023-12-29-czi-funds-pyopensci-2024.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: single
+title: "CZI Funds pyOpenSci"
+excerpt: "CZI has received funding from the Chan Zuckerberg Foundation"
+author: "Leah Wasser"
+permalink: /blog/czi-funds-pyOpenSci-2024.html
+header:
+  overlay_color: "#666"
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+toc: true
+comments: true
+---
+
+## A bright 2024 is ahead for pyOpenSci
+
+We are thrilled to announce that pyOpenSci has received 2 years of funding to cover core operations from the [Chan Zuckerberg Initiative (CZI)](https://chanzuckerberg.com/). These CZI funds will be used to continue critical pyOpenSci work that:
+
+* supports Python open source maintainers in developing the usable and maintainable scientific software the enables open science
+* helps scientists navigate a complex Python packaging ecosystem
+* diversifies the scientific Python open source community
+
+
+## Open science builds trust and enables more people to participate
+
+Open and reproducible science builds trust and accelerates research and
+discovery. Open science supports scientific research that is both transparent and
+reusable. Free and open source software is critical to open science as it ensures that
+the analyses of research data are broadly accessible. To build truly open research workflows, scientists need to use free and open source software (FOSS). FOSS removes the barriers that licenses and other fees may create, particularly for undeserved communities.
+
+### Broad inclusion is critical for both open source and open science
+
+Broad inclusion of individuals in the open source ecosystem which has been found to be [even less diverse then the broader tech community](https://www.wired.com/2017/06/diversity-open-source-even-worse-tech-overall/) is also critical to pyOpenSci's mission. pyOpenSci wants to empower everyone to participate in both the development and use of open source software. This
+empowerment enables open science to reach its full potential.
+
+### Open source maintainers need more support
+Despite the importance of open source software and inclusion to fundamental
+open science principles, open source maintainers do not get the support they
+need. Maintainers need both institutional and community support in learning
```

Comment:
```suggestion
Maintainers need both institutional and community support in learning
and engaging in open source software development, as well as in maintaining tools and interacting
with the broad user base that uses these tools in support of open science.
```

Reply:
can you help me understand the intended change here @kierisi so i can apply it?

Reply:
oh! yes, for some reason it wouldn't let me select more than a single line for review. so what's showing up in green are my suggested changes, but it's not showing up as a diff

Reply:
oh interesting! maybe we can look at that together to see why :) sometimes the github gui can be a bit quirky!